easybuild/easyconfigs/m/MView/MView-1.67-GCC-11.3.0.eb

```
@@ -0,0 +1,26 @@
+easyblock = "Tarball"
+
+name = 'MView'
+version = "1.67"
+
+homepage = 'https://desmid.github.io/mview/'
+description = """ MView reformats the results of a sequence database search or a 
+ multiple alignment, optionally adding HTML markup."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = [('http://sourceforge.net/projects/bio-mview/files/bio-mview/mview-%(version)s/', 'download')]
+sources = [SOURCELOWER_TAR_GZ]
+
```

```suggestion
checksums = ['e5bac78960f8f6c091b2f7ea8a3c6075e9bea5a062391fd3e1e44fca14025e46']

```

Missing checksum, generated with `eb MView-1.67-GCC-11.3.0.eb --inject-checksums`